### PR TITLE
Reducción de deuda técnica

### PR DESCRIPTION
**Pull request by andrea-sanchez**
## Implementación en R:

- [x] Corre una prueba dummy con testthat en GitHub Actions
- [x] Evalúa cobertura con covr en GitHub Actions
- [x] Realiza análisis estático (despeluza) con lintr en GitHub Actions

## Despeluzamiento:

- [x] src/01_create_parameter_logistic_model_LAAL.R:8:18: style: Only use double-quotes. results_path <- ('data/processed/')
- [x] src/01_create_parameter_logistic_model_LAAL.R:26:1: style: Variable and function name style should be snake_case. calculador_ROC <- ROC$new()
- [x] src/01_create_parameter_logistic_model_LAAL.R:97:1: style: Lines should not be more than 100 characters. lapply(.SD[, sapply(.SD, is.numeric), with = FALSE], mean, na.rm = T),
- [x] src/01_create_parameter_logistic_model_LAAL.R:104:1: style: Lines should not be more than 100 characters. # Se identifican los `fields` en los metadata (son las variables de los Datos) con unidades definidas ya que estos (campos) describen a las variables morfométricas.
- [x] src/01_create_parameter_logistic_model_LAAL.R:109:1: style: Lines should not be more than 100 characters. #listaVariablesMorfometricas <- tolower(paste0(large_names_morphometry[1], paste(", ", large_names_morphometry[2 (n_morphometric_variables-1)], collapse = ""), " y ", large_names_morphometry[n_morphometric_variables]))
- [x] src/01_create_parameter_logistic_model_LAAL.R:109:4: style: Commented code should be removed. #listaVariablesMorfometricas <- tolower(paste0(large_names_morphometry[1], paste(", ", large_names_morphometry[2:(n_morphometric_variables-1)], collapse = ""), " y ", large_names_morphometry[n_morphometric_variables]))
- [x] src/01_create_parameter_logistic_model_LAAL.R:123:68: style: Commas should always have a space after.
  normalized_data$sexo <- averaged_data[!is.na(averaged_data$peso),]$sexo
- [x] src/01_create_parameter_logistic_model_LAAL.R:135:36: style: Trailing whitespace is superfluous. # Aplicamos el método _stepwise_.  
- [x] src/01_create_parameter_logistic_model_LAAL.R:144:70: style: Commas should always have a space after. normalized_data$darvic <- averaged_data[!is.na(averaged_data$peso),]$darvic
- [x] src/01_create_parameter_logistic_model_LAAL.R:154:5: style: Trailing whitespace is superfluous. ## 
- [x] src/01_create_parameter_logistic_model_LAAL.R:183:1: style: Lines should not be more than 100 characters. model_table$min_normalization_parameters[i, i_pair_normalization] <- min_normalized_data[i_pair_normalization]
- [x] src/01_create_parameter_logistic_model_LAAL.R:184:1: style: Lines should not be more than 100 characters. model_table$max_normalization_parameters[i, i_pair_normalization] <- max_normalized_data[i_pair_normalization]
- [x] src/01_create_parameter_logistic_model_LAAL.R:194:1: style: Lines should not be more than 100 characters. dimorphism_model_albatross$load_parameters("data/processed/parametros_modelo_logistico_laal_ig.json")
- [x] src/01_create_parameter_logistic_model_LAAL.R:196:44: style: Only use double-quotes. y_test <- ifelse(validation_data$sexo == 'M', 1, 0)
- [x] src/01_create_parameter_logistic_model_LAAL.R:212:1: style: Lines should not be more than 100 characters. colnames(model_table$standard_error) <- c("stdErrIntercept", "stdErrlongitudCraneo", "stdErrAltoPico",
- [x] src/01_create_parameter_logistic_model_LAAL.R:223:1: style: Lines should not be more than 100 characters. model_table$min_normalization_parameters <- model_table$min_normalization_parameters[, no_intercept_variables]
- [x] src/01_create_parameter_logistic_model_LAAL.R:225:1: style: Lines should not be more than 100 characters. "minLongitudPico", "minTarso", "minAnchoCraneo")
- [x] src/01_create_parameter_logistic_model_LAAL.R:227:1: style: Lines should not be more than 100 characters. model_table$max_normalization_parameters <- model_table$max_normalization_parameters[, no_intercept_variables]
- [x] src/01_create_parameter_logistic_model_LAAL.R:229:1: style: Lines should not be more than 100 characters. "maxLongitudPico", "maxTarso", "maxAnchoCraneo")
- [x] src/01_create_parameter_logistic_model_LAAL.R:248:43: style: Commas should always have a space after. filtered_table <- completed_table[!row_na,]
- [x] src/01_create_parameter_logistic_model_LAAL.R:254:24: style: Only use double-quotes. paste0(results_path, 'tabla_modelos_logisticos.csv')
- [x] agrega fin de línea a la última línea de `src/01_create_parameter_logistic_model_LAAL.R`
- [x] src/02_evaluate_better_models.R:16:1: style: Variable and function name style should be snake_case. calculador_ROC <- ROC$new()
- [x] src/02_evaluate_better_models.R:28:3: style: Variable and function names should not be longer than 30 characters. max_auxiliar_normalized_parameters_table <- imported_table[i_row, 12:15]
- [x] src/02_evaluate_better_models.R:29:98: style: Commas should always have a space after. colnames(max_auxiliar_normalized_parameters_table) <- rownames(auxiliar_coefficients_table[2:5,])
- [x] src/02_evaluate_better_models.R:30:3: style: Variable and function names should not be longer than 30 characters. min_auxiliar_normalized_parameters_table <- imported_table[i_row, 8:11]
- [x] src/02_evaluate_better_models.R:31:98: style: Commas should always have a space after. colnames(min_auxiliar_normalized_parameters_table) <- rownames(auxiliar_coefficients_table[2:5,])
- [x] src/02_evaluate_better_models.R:50:33: style: Only use double-quotes. y_test <- ifelse(data$sexo == 'M', 1, 0)
- [x] src/02_evaluate_better_models.R:58:32: style: Commas should always have a space after. imported_table[minimun_error,],
- [x] src/02_evaluate_better_models.R:59:24: style: Only use double-quotes. paste0(results_path, 'tabla_mejores_modelos.csv')
- [x] Agrega fin de línea a la última ;ínea de `src/02_evaluate_better_models.R`
- [x] src/03_predict_sex.R:15:1: style: Variable and function name style should be snake_case.
calculador_ROC <- ROC$new()
- [x] src/03_predict_sex.R:18:21: warning: Avoid 1:nrow(...) expressions, use seq_len. for (i_albatross in 1:nrow(data)) {
- [x] src/03_predict_sex.R:19:28: style: Commas should always have a space after. dato <- data[i_albatross,]
- [x] src/03_predict_sex.R:29:5: style: Variable and function names should not be longer than 30 characters. max_auxiliar_normalized_parameters_table <- imported_table[i_row, 12:15]
- [x] src/03_predict_sex.R:30:1: style: Lines should not be more than 100 characters. colnames(max_auxiliar_normalized_parameters_table) <- rownames(auxiliar_coefficients_table[2:5,])
- [x] src/03_predict_sex.R:30:100: style: Commas should always have a space after. colnames(max_auxiliar_normalized_parameters_table) <- rownames(auxiliar_coefficients_table[2:5,])
- [x] src/03_predict_sex.R:31:5: style: Variable and function names should not be longer than 30 characters. min_auxiliar_normalized_parameters_table <- imported_table[i_row, 8:11]
- [x] src/03_predict_sex.R:32:1: style: Lines should not be more than 100 characters. colnames(min_auxiliar_normalized_parameters_table) <- rownames(auxiliar_coefficients_table[2:5,])
- [x] src/03_predict_sex.R:32:100: style: Commas should always have a space after. colnames(min_auxiliar_normalized_parameters_table) <- rownames(auxiliar_coefficients_table[2:5,])
- [x] src/03_predict_sex.R:34:1: style: Lines should not be more than 100 characters. minimum_value = as.list(min_auxiliar_normalized_parameters_table),
- [x] src/03_predict_sex.R:35:1: style: Lines should not be more than 100 characters. maximum_value = as.list(max_auxiliar_normalized_parameters_table)
- [x] src/03_predict_sex.R:48:1: style: Lines should not be more than 100 characters. dimorphism_model_albatross$load_parameters("data/processed/parametros_modelo_logistico_laal_ig.json")
- [x] Agrega fin de línea a la última línea de incomplete final line found on 'src/03_predict_sex.R'
- [x] src/calculator_ROC_class.R:4:1: style: Variable and function name style should be snake_case. ROC <- R6Class("ROC",
- [x] src/calculator_ROC_class.R:11:12: style: Variable and function name style should be snake_case. self$rOC <- private$roc_calculate(data)
- [x] src/calculator_ROC_class.R:12:12: style: Variable and function name style should be snake_case. self$rOC <- private$add_criterion(self$rOC)
- [x] src/calculator_ROC_class.R:34:53: style: Only use double-quotes. true_or_false <- ifelse(row$y_test == answer, 'V', 'F')
- [x] src/calculator_ROC_class.R:34:58: style: Only use double-quotes. true_or_false <- ifelse(row$y_test == answer, 'V', 'F')
- [x] src/calculator_ROC_class.R:48:26: style: Only use double-quotes. clasification <- c('F0', 'F1', 'V0', 'V1')
- [x] src/calculator_ROC_class.R:48:32: style: Only use double-quotes. clasification <- c('F0', 'F1', 'V0', 'V1')
- [x] src/calculator_ROC_class.R:48:38: style: Only use double-quotes. clasification <- c('F0', 'F1', 'V0', 'V1')
- [x] src/calculator_ROC_class.R:48:44: style: Only use double-quotes. clasification <- c('F0', 'F1', 'V0', 'V1')
- [x] src/calculator_ROC_class.R:49:7: style: Variable and function name style should be snake_case. N <- c(0, 0, 0, 0)
- [x] src/calculator_ROC_class.R:76:7: style: Variable and function name style should be snake_case. fPR <- c()
- [x] src/calculator_ROC_class.R:77:7: style: Variable and function name style should be snake_case. tPR <- c()
- [x] src/calculator_ROC_class.R:82:9: style: Variable and function name style should be snake_case. fPR <- append(fPR, private$calculate_FRP(confusion))
- [x] src/calculator_ROC_class.R:83:9: style: Variable and function name style should be snake_case. tPR <- append(tPR, private$calculate_TPR(confusion))
- [x] src/calculator_ROC_class.R:86:7: style: Variable and function name style should be snake_case. ROC <- data.frame(thresholds, tPR, fPR, p_error)
- [x] src/calculator_ROC_class.R:90:30: style: Variable and function name style should be snake_case. add_criterion = function(rOC) {
- [x] src/calculator_ROC_class.R:91:7: style: Variable and function name style should be snake_case. rOC <- rOC %>%
- [x] Agrega fin de línea a la última línea de `incomplete final line found on 'src/calculator_ROC_class.R`
- [x] Agrega fin de línea a la última línea de`src/dimorphism_model_class.R`
- [x] src/regretion_to_data_frame_coefficients_function.R:1:1: style: Variable and function names should not be longer than 30 characters. regretion2DataFrameCoefficients <- function(regression) {
- [x] src/regretion_to_data_frame_coefficients_function.R:1:1: style: Variable and function name style should be snake_case. regretion2DataFrameCoefficients <- function(regression) {
- [x] Agrega fin de línea a la última línea de `src/regretion_to_data_frame_coefficients_function.R`

### CODE REVIEW

File: src/02_evaluate_better_models.R [hunk 1]
```diff
@@ -27,17 +27,17 @@ for(i_renglon in 1:n_renglones) {
                                              parametrosModelo = tabla_coeficientes_auxiliar)
   
   readr::write_lines(jsonlite::toJSON(listaParametrosModeloNormalizacion, pretty = T),
```
Comment (nepito) [inline src/02_evaluate_better_models.R:29]: En el primer ejemplo de[ esta](https://style.tidyverse.org/syntax.html#long-lines) guía sugieren el siguinete cambio:
```suggestion
  readr::write_lines(
      jsonlite::toJSON(listaParametrosModeloNormalizacion, pretty = T), 
```

File: src/02_evaluate_better_models.R [hunk 2]
```diff
@@ -27,17 +27,17 @@ for(i_renglon in 1:n_renglones) {
                                              parametrosModelo = tabla_coeficientes_auxiliar)
   
   readr::write_lines(jsonlite::toJSON(listaParametrosModeloNormalizacion, pretty = T), 
-                     path =  "data/processed/parametros_modelo_logistico_laal_ig.json")
+                     path = "data/processed/parametros_modelo_logistico_laal_ig.json")
```
Comment (nepito) [inline src/02_evaluate_better_models.R:30]: Si aceptas la sugerencia de arrba, esto se vería:
```suggestion
       path = "data/processed/parametros_modelo_logistico_laal_ig.json"
    )
```

Comment (nepito) [inline src/02_evaluate_better_models.R:30]: Por favor encapsula en una variable el nombre del archivo `"data/processed/parametros_modelo_logistico_laal_ig.json"`

File: src/02_evaluate_better_models.R [hunk 3]
```diff
@@ -2,17 +2,17 @@ source("src/dimorphism_model_class.R")
...
 
 ruta_resultados <- "data/processed/"
-tabla_importada <- data.table::data.table(readr::read_csv(paste0(ruta_resultados,"tabla_modelos_logisticos.csv")))
+tabla_importada <- data.table::data.table(readr::read_csv(paste0(ruta_resultados, "tabla_modelos_logisticos.csv")))
 calculadorROC <- ROC$new()
 n_renglones <- nrow(tabla_importada)
 errores <- c()
 
-for(i_renglon in 1:n_renglones) {
+ for (i_renglon in 1:n_renglones) {
```
Comment (nepito) [inline src/02_evaluate_better_models.R:15]: Creo que te sobra un espacio al inicio:
```suggestion
for (i_renglon in 1:n_renglones) {
```

File: src/02_evaluate_better_models.R [hunk 4]
```diff
@@ -27,17 +27,17 @@ for(i_renglon in 1:n_renglones) {
...
-  errores <- append(errores, calculadorROC$calculateError(datos_roc,umbral))
+  errores <- append(errores, calculadorROC$calculateError(datos_roc, umbral))
 }
 
 es_error_minimo <- errores == min(errores)
 write_csv(tabla_importada[es_error_minimo,], paste0(ruta_resultados,'tabla_mejores_modelos.csv'))
 
-readr::write_lines(jsonlite::toJSON(listaParametrosModeloNormalizacion, pretty = T), path =  "data/processed/parametros_mejor_modelo_logistico_laal_ig.json")
\ No newline at end of file
+readr::write_lines(jsonlite::toJSON(listaParametrosModeloNormalizacion, pretty = T), path = "data/processed/parametros_mejor_modelo_logistico_laal_ig.json")
```
Comment (nepito) [inline src/02_evaluate_better_models.R:43]: El mismo comentario que en la linea 29:
```suggestion
readr::write_lines(
    jsonlite::toJSON(listaParametrosModeloNormalizacion, pretty = T), 
    path = "data/processed/parametros_mejor_modelo_logistico_laal_ig.json"
)
```

File: src/calculator_ROC_class.R [hunk 5]
```diff
@@ -7,35 +7,35 @@ ROC <- R6Class("ROC",
     initialize = function() {
     },
     
-    getBestThresholdAndError = function(datos){
+    getBestThresholdAndError = function(datos) {
         self$rOC <- private$calculateROC(datos)
         self$rOC <- private$addCriterion(self$rOC)
         mejoresUmbrales <- self$rOC %>% 
             group_by(criterio) %>%
             summarize(umbral = median(umbrales),
```
Comment (nepito) [inline src/calculator_ROC_class.R:15]: Creo que esto se verá mejor así:
```suggestion
            summarize(
                umbral = median(umbrales),
                error = median(pError)
            ) %>%
```

File: src/dimorphism_model_class.R [hunk 6]
```diff
@@ -2,36 +2,40 @@ library(R6)
...
+
+          ColumnaNormalizada <- as.data.frame(apply(columna, 2, normalize))
           z <- z + ColumnaNormalizada*private$getValue(variable$Variables)
-          iVarible <- iVarible+1
+          iVarible <- iVarible + 1
         }
       }
-      probabilidad <-  1 / (1+exp(-z))
+      probabilidad <- 1 / (1 + exp(-z))
       colnames(probabilidad)<- "probability"
```
Comment (nepito) [inline src/dimorphism_model_class.R:32]: Falto espacio entre en la asignación:
```suggestion
      colnames(probabilidad) <- "probability"
```

File: src/get_prediction_sex_plot_function.R [hunk 7]
```diff
@@ -1,10 +1,10 @@
 library(ggplot2)
 getPredictionSexPlot <- function(TablaDatos, prediccion) {
-  graficaPrediccion <- ggplot(TablaDatos,aes(x=darvic,y=prediccion,color=sexo,shape=sexo)) +
-  geom_point(size=5) +
+  graficaPrediccion <- ggplot(TablaDatos,aes(x = darvic, y = prediccion, color=sexo, shape = sexo)) +
+  geom_point(size = 5) +
```
Comment (nepito) [inline src/get_prediction_sex_plot_function.R:4]: La sugerencia de [aquí](https://style.tidyverse.org/ggplot2.html#long-lines-3) es la siguinete:
```suggestion
      geom_point(size = 5) +
```

File: Makefile [hunk 8]
```diff
@@ -1,4 +1,4 @@
-all: tests 
+all: reports/funcion_logistica.pdf tests
```
Comment (evaristor) [inline Makefile:1]:
```suggestion
all: tests reports/funcion_logistica.pdf
```

File: tests/testthat/tests_regretion2DataFrameCoef.R [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+          406L, 412L, 522L, 445L, 544L, 614L)
+)
+pois.mod <- glm(units ~ temp, data=icecream, 
+              family=poisson(link="log"))
+
+RegresionStep <- step(pois.mod)
+CoeficientesStep <- regretion2DataFrameCoefficients(RegresionStep)
+class(CoeficientesStep) == "data.frame"
+
+test_that("La función regresa la palabra que metimos de argumento",
```
Comment (evaristor) [inline tests/testthat/tests_regretion2DataFrameCoef.R:20]: Creo que este mensaje es incorrecto:

> La función regresa la palabra que metimos de argumento

Esta prueba no hace eso

File: tests/testthat/tests_regretion2DataFrameCoef.R [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+    {
+        expect_equal(class(CoeficientesStep), "data.frame")
+    }
+)
```
Comment (evaristor) [inline tests/testthat/tests_regretion2DataFrameCoef.R:24]: Agrega fin de línea a la última línea

File: tests/testthat/tests_regretion2DataFrameCoef.R [hunk 11]
```diff
@@ -0,0 +1,24 @@
+library(data.table) 
+
+#source("src/dimorphism_model_class.R")
+#source("src/calculator_ROC_class.R")
```
Comment (evaristor) [inline tests/testthat/tests_regretion2DataFrameCoef.R:4]: Por favor no consignes líneas con instrucciones comentadas

File: tests/testthat/tests_regretion2DataFrameCoef.R [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+source("../../src/regretion_to_data_frame_coefficients_function.R")
+
+icecream <- data.frame(
```
Comment (evaristor) [inline tests/testthat/tests_regretion2DataFrameCoef.R:7]: No entiendo este nombre en el contexto de dimorfismo. ¿Esta es una prueba de mentiritas?

File: tests/testthat/tests_regretion2DataFrameCoef.R [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+  temp=c(11.9, 14.2, 15.2, 16.4, 17.2, 18.1,
```
Comment (evaristor) [inline tests/testthat/tests_regretion2DataFrameCoef.R:8]: `temp` no cumple con nuestra [guía de estilo](https://islasgeci.github.io/guia_de_estilo/nomenclatura): `temperature` o `temporal` son mejores nombres.

> Deben ser nombres descriptivos y, en general, sin abreviaturas.

File: tests/testthat/tests_regretion2DataFrameCoef.R [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+  temp=c(11.9, 14.2, 15.2, 16.4, 17.2, 18.1, 
+         18.5, 19.4, 22.1, 22.6, 23.4, 25.1),
+  units=c(185L, 215L, 332L, 325L, 408L, 421L, 
+          406L, 412L, 522L, 445L, 544L, 614L)
+)
+pois.mod <- glm(units ~ temp, data=icecream, 
+              family=poisson(link="log"))
+
+RegresionStep <- step(pois.mod)
+CoeficientesStep <- regretion2DataFrameCoefficients(RegresionStep)
```
Comment (evaristor) [inline tests/testthat/tests_regretion2DataFrameCoef.R:17]: `CoeficientesStep` está esn _spanglish_. Por favor usa un único idoma.

File: src/regretion_to_data_frame_coefficients_function.R [hunk 15]
```diff
@@ -49,10 +49,10 @@
 #' @encoding utf-8
```
Comment (evaristor) [inline src/regretion_to_data_frame_coefficients_function.R:49]: Me parece que la documentación de este archivo no corresponde con el archivo.

File: tests/testthat/tests_regretion2DataFrameCoef.R (hunk 12 above)
Comment (andrea-sanchez) [reply tests/testthat/tests_regretion2DataFrameCoef.R:4]: Hallo! No pero sí. No es una prueba de mentiritas pero de primera mano, para revisar que hace lo que queremos, se introducen datos prueba y mi mente (madura, sensata y llena de sabiduría) sugirió que la variable se llamara `icecream`
En fin, gracias por las correcciones :)

**Comment by evaristor** [comment]
¿No corre en alguna de las versiones más recientes de nuestras imágenes: `islasgeci/base:56ca` o `islasgeci/jupyter:ff82`?
Creo que esas imágenes ya tienen Pythontex intalado, no?

**Comment by evaristor** [comment]
Ah, ya entendí. Estamos esperando que termineos el otro _PR_

**Comment by evaristor** [comment]
@andrea-sanchez : ¿Qué falta para cerrar este _PR_? ¿Cuándo planeas pedir revisión nuevamente?
Tal vez la función `change_structure` iba en otra rama `feature`?

**Comment by andrea-sanchez** [comment]
> @andrea-sanchez : ¿Qué falta para cerrar este _PR_? ¿Cuándo planeas pedir revisión nuevamente?
> Tal vez la función `change_structure` iba en otra rama `feature`?

@evaristor Sí, la función `change_structure` no debí consignarla en esta rama, ya que, su prueba (que no he podido pasar aún) es lo único que me evita poder cerrar este PR. Pretendo poder resolver esto ASAP porque no puedo consignar mucho hasta cerrar este PR.

**Comment by evaristor** [comment]
Gracias por tu respuesta @andrea-sanchez

**Comment by evaristor** [comment]
@andrea-sanchez: ¿Aquí no hay pruebas?

**Comment by andrea-sanchez** [comment]
> @andrea-sanchez: ¿Aquí no hay pruebas?

@evaristor *chan chan chan* jaja siento el suspenso detrás de esta pregunta. 
Sí pero no. Como los cambios que se hacen en esta rama son aplicar y/o seguir guías de estilo y unos cambios chicos en el archivo `Makefile` no quise agregar deuda técnica a la `master` llenándola de pruebas. 
Entonces, después de cerrar esta rama, las pruebas las meteré (no sé si esté bien dicho pero se entiende) en `features` para que sea mas fácil su revisión. Y se podría decir que sí hay pruebas que validen lo que hice en este PR porque el `Makefile`sigue corriendo re-bien :)

**Comment by evaristor** [comment]
@andrea-sanchez: Todo lo que dices me parece muy bien.
Gracias por tu respuesta.

File: src/01_create_parameter_logistic_model_LAAL.R [hunk 16]
```diff
@@ -2,15 +2,12 @@ library(data.table)
...
-source("src/evaluate_model_function.R")
-source("src/get_prediction_sex_plot_function.R")
-source("src/get_sex_probability_plot_function.R")
 source("src/regretion_to_data_frame_coefficients_function.R")
 
 directorioTDP <- ("data/raw/")
-nombreArchivoCSV <- file.path(directorioTDP,"morfometria_albatros-laysan_guadalupe.csv")
-nombreArchivoJSON <- file.path(directorioTDP,"datapackage.json")
-rutaResultados = ('data/processed/')
+rutaResultados <- ('data/processed/')
```
Comment (evaristor) [inline src/01_create_parameter_logistic_model_LAAL.R:8]: El nombre de esta variable está en [camelCase](https://en.wikipedia.org/wiki/Camel_case) pero abajo hay nombres en [snake_case](https://en.wikipedia.org/wiki/Snake_case) (como `tabla_umbral_error`).
¿Por favor podrías usar sólo CamelCase o sólo snake_case?
[La guía de estilo que seguimos](https://style.tidyverse.org/syntax.html#object-names) sugiere _snake_case_

File: src/01_create_parameter_logistic_model_LAAL.R [hunk 17]
```diff
@@ -22,17 +19,24 @@ proporcion_validacion <- 1 - proporcion_entrenamiento
 variablesParaModelo <- c("longitudCraneo", "longitudPico", "anchoCraneo", "altoPico", "tarso", "longAlaCerrada", "longAlaAbierta", "envergadura")
 nombre_columnas <- c("(Intercept)", variablesParaModelo)
 n_repeticiones <- 10
-#2000
 
-tabla_umbral_error <- data.frame(umbral <- c(),error <- c())
+
+tabla_umbral_error <- data.frame(umbral <- c(), error <- c())
```
Comment (evaristor) [inline src/01_create_parameter_logistic_model_LAAL.R:24]: El nombre de esta variable está en [snake_case](https://en.wikipedia.org/wiki/Snake_case) pero arriba hay nombres en [camelCase](https://en.wikipedia.org/wiki/Camel_case)(como `rutaResultados`).
¿Por favor podrías usar sólo CamelCase o sólo snake_case?
[La guía de estilo que seguimos](https://style.tidyverse.org/syntax.html#object-names) sugiere _snake_case_

**Comment by evaristor** [comment]
@andrea-sanchez: Por favor podrías hacer _pull_ y checar la salida de?:
```
make lint
```

**Comment by evaristor** [comment]
@andrea-sanchez ¿Por favor podrías hacer _merge_ de `IslasGECI:develop` :arrow_right: `andrea-sanchez:release/PrimerPrueba` resolviendo los conflictos?

**Comment by evaristor** [comment]
Excelente trabajo @andrea-sanchez

**Pull request merged**